setup.cfg

```diff
@@ -103,3 +161,42 @@ source-dir = docs
 #wcslib_libraries = wcs
 #gfortran_lib_dirs = None
 #gfortran_libraries = gfortran
+
+
+[coverage:run]
+omit =
```

Comment:
I'm guessing we need to replace `astropy` by `sherpa`, and probably tweak the directories?

Reply:
Yes. I don't know if we've set up any coverage testing for Sherpa at all, so this will need some thought.